package.json

```diff
@@ -61,7 +61,8 @@
     "precommit": "lint-staged",
     "prepush": "run-p lint test flow",
     "nom": "node bin/nom",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "validate": "npm-run-all --parallel lint flow"
```

Comment:
When I get all the scripts working we can document the "validate" script as a starting point after initial cloning & running `yarn install`